source/geometries/NextDemoSiPMBoard.cc

```diff
@@ -250,42 +281,62 @@ void NextDemoSiPMBoard::Construct()
 
 
   /// Placing the Holes with SiPMs & membranes inside
+  G4VPhysicalVolume* hole_phys[num_sipms_];
```

Comment:
It looks like this gives a compilation warning: `variable length arrays are a C99 feature`.
You could use an `std::vector` instead.

Reply:
After talking to @jmalbos, it seems that there is no need to define any container. You can simply assign the same pointer to the placement in each iteration of the loop.